pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: pan_joint
    joint_speed: 30.17
    motor: 
        id: 1
        init: 512
        max: 708
        min: 316

tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 30.17
    motor:
        id: 2
        init: 512
        min: 400
        max: 700
